api/graphql/makeModels.js

```diff
@@ -432,14 +432,12 @@ export default function makeModels (userId, isAdmin) {
     PostMembership: {
       model: PostMembership,
       attributes: [
-        'id'
+        'id',
+        'pinned'
```

Comment:
i recently discovered that this is not necessary at all: https://www.apollographql.com/docs/graphql-tools/resolvers.html#Default-resolver

so just remove that and you're good to merge